pipeline_dsl/concourse/pipeline.py

```diff
@@ -152,6 +152,7 @@ def main(self):
                 sys.exit(1)
 
     def __print_error(self, e: Exception):
+        assert e.__traceback__  # TODO: __traceback__ might be None, how should we handle this? print(e)?
```

Comment:
Does this still show the error even if there is no traceback?

Reply:
No, not the original error, but an `AssertionError`.
But this can only happen, if someone manually calls `__print_error`.

```
try:
    e = Exception("test")
    assert not e.__traceback__ 
    raise e
except Exception as e:
    assert e.__traceback__
```

As you can see, `raise` will set the `__traceback__`